hydra/_internal/utils.py

```diff
@@ -402,6 +402,11 @@ def _get_completion_help() -> str:
             completion_info.append(head)
             completion_info.append(plugin_cls.help(cmd).format(_get_exec_command()))
         completion_info.append("")
+
+    completion_info.append(
+        "For Zsh users check out the Hydra documentation in order to setup your shell"
+        " for using the existing Bash completion."
+    )
```

Comment:
You can add a section to the doc [here](https://hydra.cc/docs/tutorials/basic/running_your_app/tab_completion#install-tab-completion) about bash completion and link to it.

something like:
```
zsh is compatible with the Bash shell completion, see [this](link to section) for details.
```

Reply:
Cool. Will do.